tools/packchk/src/CheckFiles.cpp

```diff
@@ -241,7 +245,7 @@ bool CheckFiles::CheckDescription(RteItem* item)
 */
 bool CheckFiles::CheckUrls(RteItem* item)
 {
-  if(!item || item->GetTag() == "package") {
+  if(!item || item->GetTag() == "package" || item->GetTag().find(":", 0) != string::npos) {
```

Comment:
I believe this is a redundant change, code above at lines 44-46 effectively disables all checks for environment-specific elements